Clarify passage on defined gate invocation w/r/t "same size" (#168)
Qiskit/openqasm: source/language/gates.rst

Only built-in gate statements, calls to previously defined gates, and
timing directives can appear in ``body``. For example, it is not valid to
declare a classical register in a gate body. Looping constructs over these quantum
statements are valid.

The statements in the body
can only refer to the symbols given in the parameter or argument list,
and these symbols are scoped only to the subroutine body.

An empty body corresponds to the identity gate.

Gates must be declared before use and
cannot call themselves. The statement ``name(params) qargs;`` applies the gate,
and the variable parameters ``params`` can have any numeric type.

> The gate can be applied to any combination of qubits and quantum registers,

Reviewer: ```suggestion
The gate can be applied to any combination of qubits and quantum registers. Quantum registers must be of equivalent size. This is shown in the following example.
```

> The gate can be applied to any combination of qubits and quantum registers. Quantum registers must be of equivalent size. This is shown in the following example.

Reviewer: ```suggestion
The gate can be applied to any combination of qubit registers *of the same size*, as shown in the following example.
```

> which latter are *of the same size*, as shown in the following example.

Reviewer: Need to remove this line
